### DISCUSSION TITLE
Grid not displaying EPS (events per second) from Forwarding Sensors

### DISCUSSION THREAD

**Discussion by kvkamin** [Security-Onion-Solutions/securityonion#11110] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have attached 3 fielded sensors to the Manager Search node.  All sensors are correctly reporting Suricata and Zeek alerts. The Grid menu though, shows that the alert count is zero.  

![events_so_grid](https://github.com/Security-Onion-Solutions/securityonion/assets/7537509/d517903f-ade8-4a1c-a413-267632b22cfa)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
This is a known issue. We used to get this information from filebeat api but since 2.4 doesn't use filebeat we don't as of yet have a way to properly display it.
